looprpc/server.proto

```diff
@@ -19,6 +19,11 @@ service SwapServer {
         returns (ServerLoopInQuoteResponse);
 }
 
+enum ProtocolVersion {
```

Comment:
I'd add some comments to help readers understand what these enums mean.

Reply:
Added